frontend/src/Socket.js

```diff
@@ -2,6 +2,8 @@ import socketIOClient from 'socket.io-client';
 
 /* eslint-disable new-cap */
 const socket = new socketIOClient('mafia-hands-backend-cd.herokuapp.com');
+
+// uncomment this for backend development
```

Comment:
Change this to:
"Uncomment if you would like to use a local server to test server changes"